meeting-notes/2020-10-08-infra-wg.md

```diff
@@ -0,0 +1,71 @@
+# Falco infrastructure WG
+
+## 2020-10-08
+
+
+### MC
+
+- Spencer Krum
+
+### Who Joined
+
+- Leonardo Grasso ([leogr](https://github.com/leogr))
+- Umang Vanjara ([vanjara](https://github.com/vanjara))
+- Jonah Jones ([jonahjon](https://github.com/jonahjon))
+- Spencer Krum ([nibz](https://github.com/nibalizer))
+
```

Comment:
```suggestion
- Lorenzo Fontana ([fntlnz](https://github.com/fntlnz))
```

Reply:
Can I make a review of a review to add my name on the 17th line? :D

Reply:
Hey @maxgio92 

sorry, I did not notice that, could you fill another PR for that, please?

Reply:
Sure @leogr! Done